src/sdam/topology.ts

```diff
@@ -386,7 +386,9 @@ export class Topology extends TypedEventEmitter<TopologyEvents> {
   }
 
```

Comment:
The missing overloads in this file were revealed when I converted the monitor tests to TS.  I'm happy to revert these changes and cast in the test file instead.

Reply:
It's internal I think it's fine.